### PR TITLE
play down backstage, play up internal developer portal

### PR DESCRIPTION
**Pull request by punkle**
image

image

### CODE REVIEW

File: src/components/landing/VideoHero/VideoHero.js [hunk 1]
```diff
@@ -42,11 +42,11 @@ const VideoHero = () => {
               <div className="sm:text-center md:max-w-2xl md:mx-auto lg:col-span-7 lg:text-left">
                 <Headline className="tracking-right mt-12">
                   <span className="text-gray-900">Batteries included</span>
-                  <span className="block text-gray-900">Backstage</span>
+                  <span className="block text-gray-900">Internal Developer Portal</span>
                 </Headline>
 
                 <SubTitle
-                  text="Fully customizable with on-prem access. Scorecards and RBAC included."
+                  text="Fully customizable with on-prem access IDP built on Backstage. Scorecards and RBAC included."
```
Comment (dtuite) [inline src/components/landing/VideoHero/VideoHero.js:49]: This sentence reads weird to me. Something like this instead?

```suggestion
                  text="Built on Backstage. Fully customizable, with on-prem access. Scorecards and RBAC included."
```

File: src/components/landing/VideoHero/VideoHero.js [hunk 2]
```diff
@@ -42,11 +42,11 @@ const VideoHero = () => {
               <div className="sm:text-center md:max-w-2xl md:mx-auto lg:col-span-7 lg:text-left">
                 <Headline className="tracking-right mt-12">
                   <span className="text-gray-900">Batteries included</span>
-                  <span className="block text-gray-900">Backstage</span>
+                  <span className="block text-gray-900">Internal Developer Portal</span>
```
Comment (dtuite) [inline src/components/landing/VideoHero/VideoHero.js:45]: Can we go even better?

> [prefix] Internal Developer Portal

What's the best possible word or two that can go into `prefix`? 

Let's make a big change if we're going to make one.

Comment (punkle) [reply src/components/landing/VideoHero/VideoHero.js:45]: How about this from the vision statement:

Achieve ultimate developer effectiveness - Internal developer portal that unlocks your development potential and improves the maturity of software.

Comment (punkle) [reply src/components/landing/VideoHero/VideoHero.js]: Please review latest screenshots. Im not sure about the dot mesh thing.

**Review by jianreis — APPROVED** [review]
LGTM

File: src/pages/index.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ import PagerDutyLogo from '../../content/assets/logos/pagerduty/pagerduty.inline
 import ArgoLogo from '../../content/assets/logos/argo-cd/argo.inline.svg';
 import JiraSoftwareLogo from '../../content/assets/logos/jira/jira.inline.svg';
 
-const SEO_TITLE = 'Batteries included Spotify Backstage';
+const SEO_TITLE = 'Internal Developer Portal built on Backstage by Spotify';
```
Comment (punkle) [inline src/pages/index.js:48]: I think im learning that this is what appears in google list.

So we should think about this one too.

Comment (punkle) [reply src/pages/index.js:48]: "Out-of-the-box Internal Developer Portal powered by open-source"

**Pull request merged**